users/models.py

```diff
@@ -1,29 +1,71 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, absolute_import
 
+from datetime import datetime
+
 from django.contrib.auth.models import AbstractUser
 from django.core.urlresolvers import reverse
 from django.db import models
+from django.utils import timezone
+
+# Should be seperate but for clarity I'm putting them together to showcase our models
```

Comment:
just put them in the right places :P circle dependancies are common enough that it will take you some time to separate them. Might as well do it now.